sdk/src/client/secret/mnemonic.rs

```diff
@@ -108,8 +108,8 @@ mod tests {
             .unwrap();
 
         assert_eq!(
-            addresses[0].to_bech32("atoi"),
-            "atoi1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluehe53e".to_string()
+            &addresses[0].to_bech32("atoi").to_string(),
+            "atoi1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluehe53e"
```

Comment:
Should we `impl PartialEq<String> for Bech32Address` or something? would make a lot of things easier